plugins/katello/3.15/user_guide/remote_databases/index.md

```diff
@@ -24,6 +24,10 @@ There are two ways to deploy Foreman with remote databases:
     * restore the DBs on remote servers
     * run foreman-installer with right parameters pointing to the databases. It re-configures the databases and start all the services with new DB locations
 
+### Important note:
+
+Katello 3.15 changes the default database for Pulp from PostgreSQL v9.x to PostgreSQL v12. External databases should be upgraded to avoid conflicts.
```

Comment:
:-1: on "changes the default database for Pulp". It gives the impression MongoDB is removed, but it's not. However, it's safe to to say Foreman 2.0 has dropped support for PostgreSQL 9.x and users should upgrade to 12.

I also strongly insist that a separate note is made that Pulp 3 is installed alongside, but that lacks remote database support. Having that would have saved me from answers the same question in support.